lisa/util/__init__.py

```diff
@@ -112,6 +112,23 @@ def __str__(self) -> str:
         return message
 
 
+class RepoURLNotExistException(LisaException):
```

Comment:
It means repo does not exists, not url doesn't exist. So the simpler name is `RepoNotExsistException`.

Reply:
It can be used in other place like git repo and more.

Reply:
updated